experimental/dds/tree/src/publicContracts.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IChannelAttributes } from '@fluidframework/datastore-definitions/internal';
+
+/**
+ * {@inheritDoc @fluidframework/shared-object-base#ISharedObjectFactory."type"}
+ * @alpha
```

Comment:
should this be alpha legacy if loop will use it? we have not breaking change policy for just alpha, so look shouldn't use it

Reply:
loop should only use public or legacy

Reply:
public might also be fine since this in an experimental package

Reply:
Hmm, the tag of SharedTreeFactory/SharedTree in experimental Shared Tree seems to be alpha. But maybe just because it was old and nobody changed it.
I can make this public.

Reply:
yeah. i don't really understand how tagging intersects with package scope outside of `@fluidframework`